### PR TITLE
Added event pages for upcoming bytesize talks.

### PR DESCRIPTION
**Pull request by ewels**
Several of these are skeletons and need some content adding.

Most do not have YouTube / zoom URLs.

### CODE REVIEW

File: markdown/events/2021/bytesize-15-graphic-design.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Bytesize 15: Graphic design / pipeline diagrams"
+subtitle: Zandra Fagernäs - LMU Münich / MPI-EVA
```
Comment (apeltzer) [inline markdown/events/2021/bytesize-15-graphic-design.md:3]:
```suggestion
subtitle: Zandra Fagernäs - LMU Munich / MPI-EVA
```

Either Munich (english) or München (german) ;-)

Comment (ewels) [reply markdown/events/2021/bytesize-15-graphic-design.md:3]: I copied this from @jfy133 page! 😅

Comment (jfy133) [reply markdown/events/2021/bytesize-15-graphic-design.md:3]: :sweat_smile: Germlish...

**Review by jfy133 — APPROVED** [review]
I fixed @ZandraFagernas 's affiliation. LGTM now

**Review by apeltzer — APPROVED** [review]

**Comment by ewels** [comment]
> I fixed @ZandraFagernas 's affiliation. LGTM now

Thanks! Sorry, GitHub mobile app wouldn't let me apply the suggested changes for some reason, was getting an error..

**Pull request merged**